Changelog.md

```diff
@@ -1,10 +1,51 @@
 # eslint-plugin-no-jquery release history
 
+## v2.2.0
+* New rule: `no-error` (Ed Sanders)
+* New rule: `no-node-name` (Ed Sanders)
+* New rule: `no-camel-case` (Christophe Coevoet)
+* New rule: `no-contains` (Christophe Coevoet)
+* New rule: `no-is-empty-object` (Ed Sanders)
+* New rule: `no-is-plain-object` (Ed Sanders)
+* New rule: `no-constructor-attributes` (Ed Sanders)
+
+* Rule fix: Add `removeAttr` to `no-attr` rule (Ed Sanders)
+* Rule fix: Add `removeProp` to `no-remove-prop` rule (Ed Sanders)
+* Rule fix: Include the `$.css` utility in the `no-css` rule (Ed Sanders)
+* Rule fix: Include the `$.clone` utility in the `no-clone` rule (Ed Sanders)
+* Rule fix: Add fixer for `no-unique` (Ed Sanders)
```

Comment:
I think we should split fixers out into their own section (because the rule itself stays the same and won't suddenly break existing CI when people upgrade, which is what these are mostly about).